force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,15 +290,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
-
+        List <Integer> newInteger = new List <Integer>{1,2,3};
+        
+       // system.debug(newinteger);
+        return newInteger;
         // Add the integers 1, 2, and 3 to the list
         // add 1
         // add 2
         // add 3
 
```

Comment:
Use the add() Method to add the integers to the list 
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_collections_lists_sorting.htm